plugins/scale/scale-title-overlay.cpp

```diff
@@ -403,7 +403,7 @@ void scale_show_title_t::update_title_overlay_opt()
     {
         update_title_overlay_mouse();
         mouse_update.disconnect();
-        wf::get_core().connect_signal("pointer_motion_post", &mouse_update);
+        wf::get_core().connect_signal("pointer_motion_absolute_post", &mouse_update);
```

Comment:
absolute events happen only on Wayland/X11, so this will actually break DRM backend (which is much more important)

Reply:
Is it an issue to connect to both? (Although I agree that having this feature on X11 is probably not that important)

Reply:
You can connect to both, as long as the `mouse_update` handler does not care about its argument (because the `signal_data_t`s passed to the two signals have different types.